### PR TITLE
Create HashBamboo gentx.json

### PR DESCRIPTION
**Pull request by wzzYtu**

### CODE REVIEW

File: quark/gentxs/HashBamboo [hunk 1]
```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"HashBamboo","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"1.000000000000000000","max_rate":"1.000000000000000000","max_change_rate":"1.000000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1hsher05hsqwc3mw60k9rq0w05hqrd05jswes07","validator_address":"neutronvaloper1hsher05hsqwc3mw60k9rq0w05hqrd05j2nqff6","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"UMYCvEaLRijNlnCmseexCCNY+XtrtNoo3I9xiJWMYUk="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"826c06ab8aa8f171e5ec4542d8e5157909d9a484@161.97.187.234:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AyoGVQT1j/qeeKd9DVVXeyV9Q84OFVVuEhBDGhoGjSSl"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["8OM8O9YwZZzvDzLuFgJIMg9mEo4uoqyvndgbLaJZ7FwIKIec/TSHjmDcHMTiCkIYjMBiSdevMwrM4aq+YPCzhQ=="]}
```
Comment (albertandrejev) [inline quark/gentxs/HashBamboo:1]: Please update `rate`, `max_change_rate` and `max_rate` in accordance with the documentation.

**Review by albertandrejev — CHANGES_REQUESTED** [review]

File: quark/gentxs/HashBamboo (hunk 1 above)
Comment (wzzYtu) [reply quark/gentxs/HashBamboo:1]: {"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"HashBamboo","identity":"","website":"","security_contact":"","details":""},"commission":{"rate":"0.100000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"neutron1hsher05hsqwc3mw60k9rq0w05hqrd05jswes07","validator_address":"neutronvaloper1hsher05hsqwc3mw60k9rq0w05hqrd05j2nqff6","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"UMYCvEaLRijNlnCmseexCCNY+XtrtNoo3I9xiJWMYUk="},"value":{"denom":"untrn","amount":"1000000000"}}],"memo":"826c06ab8aa8f171e5ec4542d8e5157909d9a484@161.97.187.234:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AyoGVQT1j/qeeKd9DVVXeyV9Q84OFVVuEhBDGhoGjSSl"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""}},"signatures":["TpajLRvXr7bBdZFnMtQ9zszmq6nUUwUdEr0cYD6Ea6JSEpTeoOkHVOvm1nm+mmBUJKAsbjMqnqpAY3GpMT8Tjg=="]}

Comment (wzzYtu) [reply quark/gentxs/HashBamboo]: done

**Review by albertandrejev — APPROVED** [review]

**Comment by Medniyy** [comment]
Hey there! Some additional details should be provided in the description!
Check this:

https://github.com/neutron-org/testnets/tree/main/quark#fill-pull-request-description

**Pull request closed without merging**